mindsdb_server/interfaces/native/learn_process.py

```diff
@@ -0,0 +1,49 @@
+import torch.multiprocessing as mp
+
+
+ctx = mp.get_context('spawn')
```

Comment:
Please avoid declaring stuff outside of the class since the lifetime becomes wired with stuff like this when combined with multiprocessing and mulitple distors it has to run on.

For now just do `mp.get_context('spawn')` whenever you have to spawn a new process, it should be fairly quick to execute.